_activities/analysisfacilitiesforum.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Analysis Facilities Forum
+author: Mark Neubauer
+layout: plain
+---
+
+## Overview
+
+The Analysis Facilities (AF) Forum provides a community platform for those interested in contributing to the development of analysis facilities for use by HEP experiments, serving as a space to develop and exchange ideas. We interpret “development” in the broad sense, including the contribution of ideas from potential end users for functionality to support the analysis of HEP data, specification and planning of the facilities themselves, and technical developments needed to realize AFs. HEP experiments have their own internal processes for developing and deploying AFs; this AF Forum is intended to support and strengthen those efforts by sharing amongst a broader community the key ideas and developments. The AF Forum also collaborates with related [HSF Working Groups]({{site.baseurl}}/what_are_WGs.html), such as the [Data Analysis Working Group]({{site.baseurl}}/workinggroups/dataanalysis.html) and the [PyHEP Working Group]({{site.baseurl}}/workinggroups/pyhep.html), with strong connections to [WLCG](https://wlcg.web.cern.ch).
+
+The development of future analysis facilities is of great interest to the HEP community, with considerable recent progress. There are numerous ongoing efforts to stand up AFs that utilize new tools and techniques to help make data analysis tasks easier, more performant, and more reproducible.
+
+## What is an Analysis Facility?
+
+We loosely define an Analysis Facility as _the infrastructure and services that provide integrated data, software and computational resources to execute one or more elements of an analysis workflow. These resources are shared among members of a virtual organization and supported by that organization._
+
+This definition will likely evolve as informed by the AF Forum discussions.
+
+## Inception
+
+The idea of an AF Forum was proposed during the [Future Analysis Systems and Facilities workshop](https://indico.cern.ch/event/960587), where the need for a platform for those working toward analysis facilities within HEP experiments to develop and exchange ideas was identified. 
+
+_Kick-starters_: Diego Ciangottini (INFN, Perugia U, CMS), Alessandra Forti (Manchester, WLCG/ATLAS), Lukas Heinrich (TUM, ATLAS), Alex Held (NYU, ATLAS), Mark Neubauer (Illinois, ATLAS), Oksana Shadura (Nebraska, CMS), Nicole Skidmore (Manchester, LHCb)
+
+## How do I get involved?
+
+All are welcome to join the group and participate in the discussions! Please join the
+
+* mailing list: <hsf-AFforum-wg@googlegroups.com>
+* Mattermost:
+  * TBA
```

Comment:
Can we just set this up now? We do have an HSF Mattermost Team already, so we can add this there as a new channel.